sound/soc/sof/ipc3.c

```diff
@@ -293,6 +293,8 @@ static int ipc3_wait_tx_done(struct snd_sof_ipc *ipc, void *reply_data)
 		dev_err(sdev->dev,
 			"ipc tx timed out for %#x (msg/reply size: %d/%zu)\n",
 			hdr->cmd, hdr->size, msg->reply_size);
+
+		ipc->disable_ipc_tx = true;
```

Comment:
Just curious, when will disable_ipc_tx  be false again?

Reply:
IPC tx is protected by this flag and the FW state. If there's a FW panic and is irrecoverable, shouldnt we set the FW state to CRASHED so that further IPCs wont be sent?

Reply:
> Just curious, when will disable_ipc_tx be false again?

when the sof driver is loaded again

Reply:
> IPC tx is protected by this flag and the FW state. If there's a FW panic and is irrecoverable, shouldnt we set the FW state to CRASHED so that further IPCs wont be sent?

The SOF_FW_CRASHED will be set in my PR to support fw panic for IPC4. Thanks for advice.